src/amber/support/file_encryptor.cr

```diff
@@ -0,0 +1,37 @@
+require "./message_encryptor"
+
+module Amber::Support
+  module FileEncryptor
+    def self.global_encryption_key
+      ENV["AMBER_ENCRYPTION_KEY"]? || begin
+        if File.exists?(".encryption_key")
+          File.open(".encryption_key").gets_to_end.to_s
+        else
+          nil
+        end
+      end
+    end
+
+    def self.read(filename : String, encryption_key = global_encryption_key)
+      if encryption_key
+        encryptor = Amber::Support::MessageEncryptor.new(encryption_key)
+        encryptor.verify_and_decrypt(File.open(filename).gets_to_end.to_slice)
+      else
+        raise "Encryption key doesn't exist!"
```

Comment:
Are we using this namespacing for other exceptions: `Amber::Exception`?

I think this could work as a class name: `Amber::Exception::EncryptionKeyMissing`
Or maybe this, which will allow other Encryption key errors:
`Amber::Exception::EncryptionKey::Missing`
then
`Amber::Exception::EncryptionKey::Invalid` or whatever else makes sense.

Reply:
@marksiemers yes we have a Amber::Exception where all exceptions should inherit from.